packages/eas-cli/src/update/configure.ts

```diff
@@ -252,6 +254,71 @@ async function ensureEASUpdatesIsConfiguredNativelyAsync(
   }
 }
 
+/**
+ * Make sure EAS Build profiles are configured to work with EAS Update by adding channels to build profiles.
+ */
+
+async function ensureEASUpdateIsConfiguredInEasConfigAsync({
+  projectDir,
+}: {
+  projectDir: string;
+}): Promise<void> {
+  const easConfigPath = EasJsonAccessor.formatEasJsonPath(projectDir);
+
+  if (await fs.pathExists(easConfigPath)) {
```

Comment:
Prefer returning early from functions:
```suggestion
  if (!await fs.pathExists(easConfigPath)) {
    Log.warn(...);
    return;
```